instrumentation-tests/src/main/res/raw/route_with_road_objects_japan.json

```diff
@@ -7140,6 +7140,9 @@
               ],
               "intersections": [
                 {
+                  "merging_area": {
+                    "type": "from_left"
+                  },
```

Comment:
Did you added it by hands or Directions API generated the same route as tested previously but with merging areas?

Reply:
This is some real route and I added the "merging_area" manually, yes. We finalized the API but unfortunately we can't wait before server is released to include this functionality.